### ISSUE TITLE
input images for training need to be normalized

### ISSUE THREAD

**Issue by love2learn-7179** [Tianxiaomo/pytorch-YOLOv4#32]
I noticed that the input image for inference in do_detect() is normalized (dividing each element by 255.0), but the input images for training are not normalized. 
After normalizing the input images, fine-tuning seems working better.

Basically, I loaded the yolov4.pth and fine-tune the model for 1 epoch w/ few images (w/ no data augmentation and very small lr) to verify the functionality. Without the input normalization, it couldn't predict at all, but w/ normalization it does predict similar to the ones from converted yolov4.pth w/o fine-tuning.

**Comment by Tianxiaomo** [comment]
Modified, thanks.
╰(*°▽°*)╯
